packages/nocodb/src/lib/migrations/XcMigrationSourcev2.ts

```diff
@@ -27,6 +28,7 @@ export default class XcMigrationSourcev2 {
       'nc_018_add_meta_in_view',
       'nc_019_add_meta_in_meta_tables',
       'nc_020_add_kanban_meta_col',
+      'nc_021_rename_kanban_grp_col_id',
```

Comment:
Since we didn't published yet we can even merge these 2 migrations ( `'nc_020_add_kanban_meta_col'` and `'nc_021_rename_kanban_grp_col_id'` )